modules/social_features/social_profile/modules/social_profile_preview/config/install/core.entity_view_display.profile.profile.preview.yml

```diff
@@ -55,7 +55,12 @@ content:
     region: content
     settings: {  }
     third_party_settings: {  }
-  social_follow_user:
+  followers:
```

Comment:
Have to add hook update with updating this config by update helper for sites that already installed the `social_profile_preview` module.

Reply:
done